client/test/router_action_creator.spec.js

```diff
@@ -10,7 +10,7 @@ const SpecDispatcher = require('./utils/specs_dispatcher');
 const ActionTypes = require('../app/constants/app_constants').ActionTypes;
 
 
-describe('RouterActionCreator', () => {
+describe.skip('RouterActionCreator', () => {
```

Comment:
Maybe its missing comments : some tests have been written and added to the file as empty one to be filled in time. Maybe add a comment : "should be written" or maybe someting better describing why they are empty?